<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>main_screen</class>
 <widget class="QDialog" name="main_screen">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>680</width>
    <height>298</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Welcome screen</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(0, 0, 0);
</string>
  </property>
  <widget class="QPushButton" name="start_button">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>60</y>
     <width>113</width>
     <height>32</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 85, 0);</string>
   </property>
   <property name="text">
    <string>start</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>170</y>
     <width>121</width>
     <height>114</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="temp">
    <item row="0" column="0">
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="0" column="0">
       <widget class="QLabel" name="label_4">
        <property name="styleSheet">
         <string notr="true">
font: 75 italic 14pt &quot;Source Code Pro&quot;;
color: rgb(72, 145, 217);

background-color: rgba(255, 255, 255, 0);</string>
        </property>
        <property name="text">
         <string>Temp</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLabel" name="label_6">
        <property name="styleSheet">
         <string notr="true">font: italic 16pt &quot;PibotoLt&quot;;


color: rgb(72, 145, 217);

background-color: rgba(255, 255, 255, 0);</string>
        </property>
        <property name="text">
         <string>C</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="dive_temp">
      <property name="styleSheet">
       <string notr="true">font: 75 48pt &quot;DejaVu Sans&quot;;
background-color: rgba(255, 255, 255, 0);
color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>0</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>30</y>
     <width>121</width>
     <height>114</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="Depth">
    <item row="0" column="0">
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="label_3">
        <property name="styleSheet">
         <string notr="true">
font: 75 italic 14pt &quot;Source Code Pro&quot;;
color: rgb(72, 145, 217);
</string>
        </property>
        <property name="text">
         <string>Depth</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLabel" name="label_5">
        <property name="styleSheet">
         <string notr="true">font: italic 16pt &quot;PibotoLt&quot;;


color: rgb(72, 145, 217);

background-color: rgba(255, 255, 255, 0);</string>
        </property>
        <property name="text">
         <string>m</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="dive_depth">
      <property name="styleSheet">
       <string notr="true">font: 75 48pt &quot;DejaVu Sans&quot;;
background-color: rgba(255, 255, 255, 0);
color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>0</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>30</y>
     <width>311</width>
     <height>111</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="time">
    <item row="1" column="0">
     <widget class="QLabel" name="label_2">
      <property name="styleSheet">
       <string notr="true">
font: 75 italic 14pt &quot;Source Code Pro&quot;;
color: rgb(72, 145, 217);

background-color: rgba(255, 255, 255, 0);</string>
      </property>
      <property name="text">
       <string>Dive Time</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0" rowspan="2">
     <widget class="QLabel" name="dive_time">
      <property name="styleSheet">
       <string notr="true">font: 75 48pt &quot;DejaVu Sans&quot;;
background-color: rgba(255, 255, 255, 0);
color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>00:00:00</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="end_button">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>110</y>
     <width>113</width>
     <height>32</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 85, 0);</string>
   </property>
   <property name="text">
    <string>End</string>
   </property>
  </widget>
  <widget class="QPushButton" name="back_button">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>160</y>
     <width>113</width>
     <height>32</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 85, 0);</string>
   </property>
   <property name="text">
    <string>Back</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
